pkg/v19/key/key.go

```diff
@@ -62,6 +62,22 @@ func ClusterOrganization(clusterGuestConfig v1alpha1.ClusterGuestConfig) string
 	return clusterGuestConfig.Owner
 }
 
+// CommonAppSpecs returns apps installed for all providers.
+// Note: When adding app specs you also need to add the chart name to the
+// desired state tests in the chartconfig and configmap services.
+func CommonAppSpecs() []AppSpec {
+	return []AppSpec{
+		{
+			App:             "kube-state-metrics",
+			Catalog:         "giantswarm",
+			Chart:           "kube-state-metrics-app",
+			Namespace:       metav1.NamespaceSystem,
+			UseUpgradeForce: true,
+			Version:         "0.5.0",
+		},
```

Comment:
We keep using key structs for now. But at least the app names and versions will be moving to the release CRs when we're ready.